src/iris/api.py

```diff
@@ -1092,6 +1098,21 @@ def on_get(self, req, resp):
 
         query = plan_query % ', '.join(plan_columns[f] for f in fields)
 
+        connection = db.engine.raw_connection()
+        cursor = connection.cursor(db.ss_dict_cursor)
+
+        plan_target = req.params.pop('target', None)
```

Comment:
You should be able to integrate this into `gen_where_filter_clause`. That gets us `startswith`, `contains`, and `endswith` for free. We'll probably want to use `contains` for integrating this into the UI anyway.

Reply:
implemented